history.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+type HistoryValue interface{}
+
+type History struct {
+	ItemID string       `json:"itemid"`
+	Value  HistoryValue `json:"value"`
+	Clock  string       `json:"clock"`
+}
+
+type ItemHistory struct {
+	Item    Item
+	History History
+}
+
+func (history *History) String() string {
+	return fmt.Sprint(history.Value)
+}
+
+func (history *History) date() time.Time {
+	date, _ := strconv.ParseInt(history.Clock, 10, 64)
+	return time.Unix(date, 0)
+}
+
+func (history *History) DateTime() string {
```

Comment:
please check that result of `date()` is not `0`, in this case we need to return `-`